src/main/java/com/github/checkstyle/generatepatchfile/GeneratePatchFileLauncher.java

```diff
@@ -45,6 +48,14 @@ public static void main(String[] args) throws Exception {
         final GeneratePatchFile generatePatchFile =
                 new GeneratePatchFile(repoPath, testerPath, checkstyleRepoPath,
                         checkstyleBranch, baseConfigFile, patchConfigFile);
-        generatePatchFile.generatePatch(Integer.parseInt(args[6]));
+        final String commitParam = args[6];
+        if (commitParam.length() == 1) {
```

Comment:
You can't use args[6] > 9 in this case. Please do `if (commitParam.matches("(0|[1-9]\\d*)"))` and fix README description accordingly.

Reply:
done